Ea update azure ra (#1362)
hashicorp/terraform-website: content/source/docs/enterprise/before-installing/reference-architecture/azure.md

must be specified during the Terraform Enterprise installation for application data to
be stored securely and redundantly away from the Azure VMs running the
Terraform Enterprise application. This Azure Blob Storage container must be in the same
region as the VMs and Azure Database for PostgreSQL instance.

> It is recommended the virtual network containing the Terraform Enterprise servers be configured with a

Reviewer: ```suggestion
We recommend that the virtual network containing the Terraform Enterprise servers be configured with a
```

encryption](https://docs.microsoft.com/en-us/azure/storage/common/storage-service-encryption)
by Azure Blob Storage if required by your security policy.

> Due to support within the product, and increased durability, for a single-region deployment, we recommend using zone-redundant storage (ZRS) which synchronously writes across three Azure availability zones in the region.  For a multi-region deployment, use geo-zone-redundant storage (GZRS) for added region redundancy.

Reviewer: What does "due to support within the product" mean?

Author: Clarified.
